### PR TITLE
Added missing org for IncrementFi

### PR DESCRIPTION
**Pull request by franklywatson**

### CODE REVIEW

File: data/ecosystems/f/flow.toml [hunk 1]
```diff
@@ -75,6 +75,7 @@ github_organizations = [
   "https://github.com/FlowFans-Learning",
   "https://github.com/letthemusicpay",
   "https://github.com/LiquidityTokenLabs",
+  "https://github.com/orgs/IncrementFi",
```
Comment (eherrerosj) [inline data/ecosystems/f/flow.toml:78]: can you replace this org URL with adding IncrementFi as a subecosystem of Flow (see list at the beginning of this same file)?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Added a comment

File: data/ecosystems/f/flow.toml
Comment (btspoony) [inline data/ecosystems/f/flow.toml:1]: I found that IncrementFi already exists in the sub-ecosystems, but just missing some repos:
https://github.com/electric-capital/crypto-ecosystems/blob/master/data/ecosystems/i/increment-finance.toml

Comment (eherrerosj) [reply data/ecosystems/f/flow.toml:1]: @franklywatson could you please adjust the PR to add the missing repos to the existing toml?

**Review by eherrerosj — CHANGES_REQUESTED** [review]

**Review by eherrerosj — APPROVED** [review]

**Pull request merged**